docs/source/cc_basic.md

```diff
@@ -0,0 +1,209 @@
+# Running Chaincode as a Service on the Test Network
+
+The Fabric v2.4.1 chaincode-as-a-service feature is a novel, practical approach to running smart contracts. By comparison, the earlier method required the peer to orchestrate the complete lifecycle of the chaincode. This required the peer to have access to the Docker Daemon to create images, and to start containers. Java, Node.js and Go chaincode frameworks were explicitly known to the peer, including how they should be built and started.
```

Comment:
wouldn't say it was particular novel :-)

Reply:
ok sure ;) ... I think you had useful and practical, which seem similar.